__tests__/business-text/business-text.mutation.test.js

```diff
@@ -11,14 +10,18 @@ const {
   updatedBusinessText,
   notExistBusinessTextId,
 } = require('./business-text.variables');
-require('dotenv').config();
+const { setupApp } = require('../helper-functions');
 
 let businessText = null;
 let businessTextId = '';
+let operations;
 
-describe('Business text mutations test', () => {
+describe('Business page queries', () => {
+  beforeAll(async () => {
+    operations = await setupApp();
+  });
   test('#1 should add business text to database', async () => {
```

Comment:
Please, remove `#1` from the description string

Reply:
Fixed